dart/lib/src/scope.dart

```diff
@@ -40,12 +40,6 @@ class Scope {
 
   set span(ISentrySpan? span) {
     _span = span;
-
```

Comment:
@lColinDl 

> /home/runner/work/sentry-dart/sentry-dart/dart/lib/src/scope.dart:39:20 Avoid wrapping fields in getters and setters just to be "safe".
>   Notice: See https://dart-lang.github.io/linter/lints/unnecessary_getters_setters.html to learn more about this problem.

We can ignore the linter otherwise analyzer will fail.